Mathlib/Analysis/Normed/Ring/Seminorm.lean

```diff
@@ -344,3 +381,14 @@ def RingSeminorm.toRingNorm {K : Type*} [Field K] (f : RingSeminorm K) (hnt : f
 def normRingNorm (R : Type*) [NonUnitalNormedRing R] : RingNorm R :=
   { normAddGroupNorm R, normRingSeminorm R with }
 #align norm_ring_norm normRingNorm
+
+
+/-A multiplicative ring norm with value in the rationals satisfies `f n ≤ n` for every `n : ℕ`-/
+lemma MulRingNorm_nat_le_nat (n : ℕ) (f : MulRingNorm ℚ) : f n ≤ n := by
+  induction' n with n hn
```

Comment:
Can you use `induction` instead of `induction'`? The latter sooner or later will disappear.